### DISCUSSION TITLE
Enable custom domain

### DISCUSSION THREAD

**Discussion by CXRom** [appwrite/appwrite#1071] [Q&A] [answered]
I'm trying to use a custom domain in my application but after I successfully verified the domain this show in TLS column `In progress` do I need to make an additional configuration or how can I see if something is throwing an error?

---
Appwrite Version: 0.7.2.145

**Comment by kodumbeats** [comment, marked as answer]
Hi @CXRom 👋🏻 
You can find more information in the logs of the certificates worker:
`docker-compose logs appwrite-worker-certificates`

If that doesn't lead anywhere, we've got some troubleshooting tips in our docs that may help:
https://appwrite.io/docs/debugging

Also, here's a link to our guide on custom domains: 
https://appwrite.io/docs/custom-domains

🤞🏻 Let us know how it goes!

**Reply by PtradeLLC** [reply]
Seems like you know what you are talking about, how do I find DNS info for appwrite to plug into Godaddy. I installed Appwrite locally, trying to add custom domain, but need info for the DNS which is not on the `Create Domain` page.
